### DISCUSSION TITLE
Errors occurs while running the code in terminal

### DISCUSSION THREAD

**Discussion by thebikerholic** [bitcraze/discussions#148] [Q&A]
![Screenshot from 2022-07-16 13-53-52](https://user-images.githubusercontent.com/101856934/179347240-f417b09e-4a0e-4289-a5ab-cda58377aad5.png)
please help to recover this problem.what is the problem means

**Comment by knmcguire** [comment]
We need more information about your problem:

- Did you go through this tutorial first: https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/
- If so, After ['Update Crazyflie and Aideck firmware'](https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/#update-crazyflie-and-aideck-firmware), do you see 'ESP32: I (910) SYS: Initialized' ?
- Which revision of the AIdeck do you have (underneath the deck) and if the AIdeck is Rev. A-C, did you managed to do this step: https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/#gap8-bootloader-for-older-aidecks-revisions-a-c-
